lib/awful/hotkeys_popup/keys/qutebrowser.lua

```diff
@@ -0,0 +1,186 @@
+---------------------------------------------------------------------------
+--- Qutebrowser hotkeys for awful.hotkeys_widget
+--
+-- @author Simon Désaulniers sim.desaulniers@gmail.com
+-- @copyright 2017 Simon Désaulniers
+-- @module awful.hotkeys_popup.keys.qutebrowser
+---------------------------------------------------------------------------
+
+local hotkeys_popup = require("awful.hotkeys_popup.widget")
+
+local qutebrowser_rule_any = {class={"__main__.py", "qutebrowser"}}
```

Comment:
This will match any `__main__.py` class, no?
When is `__main__.py` used in qutebrowser?
I get `WM_CLASS(STRING) = "qutebrowser", "qutebrowser"` for the main window.